Lib/packTab/__init__.py

```diff
@@ -71,6 +71,8 @@
 
 __all__ = ['Code', 'pack_table', 'pick_solution']
 
+__version__ = "0.1.0"
```

Comment:
@behdad you bump this version string here before cutting a new release. Every release on the Python Package Index must have unique version strings (can't re-upload same version twice).